src/app/components/builder-bg-selector/builder-bg-selector.component.scss

```
@@ -0,0 +1,36 @@
+.background-select-wrapper {
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+
+  .background-select-title {
+    margin-top: 0.5rem;
+  }
+
+  .background-select-group {
+    display: flex;
+    flex-direction: row;
+
+    .background-select-option {
+      height: 2.2rem;
+      width: 2.2rem;
```

посмотри в сторону использования всех способностей scss
```suggestion
     .background-select-group {
       display: flex;
       flex-direction: row;

       &__option {  //это класс будет background-select-group__option
         height: 2.2rem;
         width: 2.2rem;
        }
```